### ISSUE TITLE
Browse mode: resize drag handle does not have keyboard support

### ISSUE THREAD

**Issue by joedolson** [WordPress/gutenberg#47336]
### Description

The drag handle to resize the navigation and editor panes in full site editing is focusable and labeled appropriately, but does not respond to arrow key presses to adjust sizing. If they are controlled by a key other than arrow keys, that needs to be communicated; but I'd also consider that to be a strange choice.

### Step-by-step reproduction instructions

1. Go to Site Editing
2. Select 'Manage All Templates' to set focus position at end of navigation pane.
3. Press tab to reach the drag handle.
4. Attempt to adjust the pane sizes using arrow keys. 

### Environment info

Gutenberg 15

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by annezazu** [comment]
cc @youknowriad as a heads up!

**Comment by Mamaduka** [comment]
Probably resolved by #47176.

**Comment by youknowriad** [comment]
Yes this has been solved. I'm closing the issue, let me know if there's anything missing.

**Comment by joedolson** [comment]
So many very similar words, and yet I didn't find it. I'll admit, I wasn't aware that I would need to also search pull requests that may not be associated with an issue. Glad this is handled!
